UnitsNet.NumberExtensions/UnitsNet.NumberExtensions.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- NuGet properties -->
+  <PropertyGroup>
+    <PackageId>UnitsNet.NumberExtensions</PackageId>
+    <Version>4.44.0</Version>
+    <Authors>Andreas Gullberg Larsen</Authors>
```

Comment:
If you wish, I think you should put your name as author next to mine. You made this nuget all yourself and did a super job! I can only take credit for the old code it was inspired by.

> A comma-separated list of packages authors, matching the profile names on nuget.org

https://docs.microsoft.com/en-us/nuget/reference/nuspec#authors

Reply:
That's excellent suggestion, I've added my colleague's and my name there.
Thanks.